priv/schema/rabbit.schema

```diff
@@ -1067,13 +1067,6 @@ end}.
 %% NB: Change these only if you understand what you are doing!
 %%
 
-%% Explicitly enable/disable hipe compilation.
-%%
-%% {hipe_compile, true},
-
-{mapping, "hipe_compile", "rabbit.hipe_compile",
```

Comment:
This will break existing installations that have `hipe_compile` in the config file. We cannot accept this. Making it a no-op is fine but this would make nodes fail to start.

Reply:
I've reverted this & added a note on deprecation.